algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -290,16 +290,29 @@ public Integer getAroundPrecision() {
     }
 
     private static final String KEY_AROUND_RADIUS = "aroundRadius";
+    public static final Integer RADIUS_ALL = Integer.MAX_VALUE;
```

Comment:
Good catch, there's no valid reason for boxing here! Updated to use `int` in 74bdca9.